tests/integration/test_loader_and_execution_options.py

```diff
@@ -81,6 +74,12 @@ class CountryRepository(SQLAlchemySyncRepository[UUIDCountry]):
         usa_country_3 = star_country_repo.get_one(name="United States of America")
         assert len(usa_country_3.states) == 2
 
+        si1_country_repo = CountryRepository(session=db_session)
+        usa_country_1 = si1_country_repo.get_one(
+            name="United States of America",
+            load=[noload(UUIDCountry.states)],
+        )
+        assert len(usa_country_1.states) == 0
```

Comment:
If you move this to the first test here, then this block will pass but all the others fail.